stress-example/nlp_search/flows/index.yml

```diff
@@ -0,0 +1,27 @@
+!Flow
+version: '1'
+pods:
+  - name: segmenter
+    show_exc_info: true
+    uses: pods/segment.yml
+    read_only: true
+  - name: encoder
+    show_exc_info: true
+    uses: pods/encoder.yml
+    parallel: $JINA_PARALLEL
+    timeout_ready: 600000
+    read_only: true
+  - name: chunk_idx
+    show_exc_info: true
+    uses: pods/chunk.yml
+    shards: $JINA_SHARDS
```

Comment:
`shards` is the same as `parallel`. Maybe specify which ones are used when in the naming

`JINA_SHARDS_CHUNK_IDX` and `JINA_SHARDS_ENCODER` etc. And only use one of the names, either `shard` or `parallel`-based